storage/object/api-cli/bucket-operations.mdx

```diff
@@ -384,7 +388,7 @@ x-amz-date: 20190411T100211Z
 </ListVersionsResult>
 ```
 
-**CLI Ccmmand** 
+**CLI Command** 
```

Comment:
```suggestion
**CLI command** 
```

Reply:
it's a title without a verb, so I'll keep all caps